src/containers/NonBitGoRecoveryCoin/OpethTokenForm.tsx

```diff
@@ -0,0 +1,167 @@
+import { Form, FormikHelpers, FormikProvider, useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import * as Yup from 'yup';
+import {
+  Button,
+  FormikPasswordfield,
+  FormikSelectfield,
+  FormikTextarea,
+  FormikTextfield,
+} from '~/components';
+
+const validationSchema = Yup.object({
+  backupKey: Yup.string().required(),
+  gasLimit: Yup.number()
+    .typeError('Gas limit must be a number')
+    .integer()
+    .positive('Gas limit must be a positive integer')
+    .required(),
+  gasPrice: Yup.number()
+    .typeError('Gas price must be a number')
+    .integer()
+    .positive('Gas price must be a positive integer')
+    .required(),
+  krsProvider: Yup.string()
+    .oneOf(['keyternal', 'bitgoKRSv2', 'dai'])
+    .label('Key Recovery Service'),
```

Comment:
krs provider is not available for opeth and arbeth, so it can be removed

Reply:
addressed